tensorflow/python/ops/array_ops.py

```diff
@@ -1273,6 +1277,57 @@ def transpose(a, perm=None, name="transpose"):
     return ret
 
 
+def framesig(sig, frame_len, frame_step, parallel=False, name="framesig"):
```

Comment:
let's not allow parallel; at least to start with. h i believe parallel_stack does not support backprop.